docs/sdks/server-sdks/dart/initialization.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+import ApiOptionRef from "@site/src/components/ApiOptionRef";
+
+The Eppo Dart SDK is easy to initialize while offering robust customization options, making it adaptable to various use cases such as custom caching requirements and ultra-low-latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one.
+
+```dart
+import 'package:eppo_sdk/eppo_sdk.dart';
+
+final client = new EppoClient(sdkKey: '<SDK_KEY>');
+
+// Wait for the client to be ready before making assignments
+await client.whenReady();
+```
+
+## Use the SDK instance
+
+After initialization, you can use the SDK instance to assign a variation to a subject using the assignment methods.
+
+```dart
+// Create a subject with the current user's information
+final subject = Subject(getCurrentUserId())
+  ..stringAttribute('country', user.country)
+  ..stringAttribute('device', user.device);
+
+// Make an assignment
+final variation = client.stringAssignment(
+  'my-feature',
+  subject,
+  'default'
+);
+```
+
+## Advanced Configuration
+
+Basic initialization is great for most use cases, but the SDK provides options that you can use during initialization to customize the behavior of the SDK.
+
+### Initialization Options
+
+How the SDK fetches, serves, and caches experiment configurations is configurable via the `EppoClient` constructor:
+
+<ApiOptionRef 
+  name="PollingIntervalInMillis"
+  type="int"
+  defaultValue="30000"
+>
+
+The interval in milliseconds between polling attempts to fetch updated configurations from the Eppo API.
+
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="PollingJitter"
+  type="long"
+  defaultValue="5000"
+>
+
+The maximum amount of time in milliseconds to add to the polling interval.
+
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="logger"
+  type="AssignmentLogger"
+  defaultValue="null"
+>
+
+A callback that sends each assignment to your data warehouse. Required for experiment analysis.
```

Comment:
This should be an object instance of the AssignmentLogger class instead of a callback